### ISSUE TITLE
Gitmoji preset expression is not extracting the type correctly from commit message

### ISSUE THREAD

**Issue by kaubry** [mooltiverse/nyx#122]
Using the "extended" presets and using the gitmoji convention,  the changelog is always empty when creating a release when following gitmoji types.

Given the following git message:  `:sparkles: Add rocket`

The presets configuration extract the type as `sparkles` but changelogs expect `:sparkles:`.

Preset:
https://github.com/mooltiverse/nyx/blob/9f4553651257ce1ec147c96f983fab0ba6954aa0/modules/java/main/src/main/java/com/mooltiverse/oss/nyx/configuration/presets/CommitMessageConventions.java#L42

Changelog:
https://github.com/mooltiverse/nyx/blob/9651403671c261aea4b3ea382a775289a65f6bd9/modules/java/main/src/main/java/com/mooltiverse/oss/nyx/configuration/presets/Changelogs.java#L32,  the expected type is `:sparkles:` but is currently `sparkles`

The above issue applies for all the Gitmoji commit types.

**Comment by kaubry** [comment]
Hi @flelli ,

Sorry, the issue is not very clear how I explained it. I just updated the description. Let me know if it's clearer.
The suggested fix would be to change the regex in CommitMessageConventions.java to:

```java
setExpression("(?m)^(?:<type>[a-zA-Z0-9_]+:)( (?<title>.+))?$(?s).*"); 
```

which would include the colon in the type group.

I'm happy to raise a PR tomorrow if you are aligned with the solution.

**Comment by flelli** [comment]
I just published a fix as version [1.2.3-hotfix122.1](https://github.com/mooltiverse/nyx/releases/tag/1.2.3-hotfix122.1).
Can you please give it a try before I merge to main?

**Comment by kaubry** [comment]
I tested it locally, and it seems to work. 
Thank you @flelli

**Comment by flelli** [comment]
Merged into main and available in stable release [1.2.3](https://github.com/mooltiverse/nyx/releases/tag/1.2.3).
